pkg/exchange/okex/exchange.go

```diff
@@ -304,7 +304,7 @@ func (e *Exchange) CancelOrders(ctx context.Context, orders ...types.Order) erro
 }
 
 func (e *Exchange) NewStream() types.Stream {
-	return NewStream(e.client)
+	return NewStream(e.key, e.secret, e.passphrase)
```

Comment:
why passing secret and passphrase into the stream? can you pass the client instead?